dev/geth-genesis-local.json

```diff
@@ -17,17 +17,22 @@
         "ethash": {},
         "astriaRollupName": "astria-test-chain-1",
         "astriaOverrideGenesisExtraData": true,
-        "astriaSequencerInitialHeight": 2,
-        "astriaSequencerAddressPrefix": "astria",
-        "astriaCelestiaInitialHeight": 2,
-        "astriaCelestiaHeightVariance": 10,
-        "astriaBridgeAddresses": [],
-        "astriaBridgeSenderAddress": "0x0000000000000000000000000000000000000000",
-        "astriaFeeCollectors": {
-            "1": "0xaC21B97d35Bf75A7dAb16f35b111a50e78A72F30"
-        },
-        "astriaEIP1559Params": {
-            "1": { "minBaseFee": 0, "elasticityMultiplier": 2, "BaseFeeChangeDenominator": 8 }
+        "astriaForks": {
+            "testLaunch": {
+                "height": 1,
+                "feeCollector": "0xaC21B97d35Bf75A7dAb16f35b111a50e78A72F30",
+                "eip1559Params": { "minBaseFee": 0, "elasticityMultiplier": 2, "BaseFeeChangeDenominator": 8 },
+                "sequencer": {
+                    "chainId": "sequencer-test-chain-0",
```

Comment:
is this used for anything in particular? it doesn't seem to be used apart from being checked to be non-empty. same for celestia chain id